Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -293,7 +293,7 @@
 
 - type: cargoBounty
   id: BountyPercussion
-  reward: 15000
+  reward: 2000
```

Comment:
Sci can't print percussion instruments which means you once again have to buy them to ever complete this bounty. You need 2 crates at 2.5k each, you make a loss on fufilling this bounty

Reply:
You can:

> order it
> find it at salvage wrecks
> ask musician nicely to give you them

Reply:
ordering it doesnt make a profit
salv dependent = bad (refer to my statement on the figurines)
musician is map dependent and/or role dependant (some maps have percussion, some don't! Some musicians bring percussion instruments in, most dont)